src/main/java/lotto/Application.java

```diff
@@ -1,7 +1,67 @@
 package lotto;
 
+import java.util.List;
+import java.util.stream.Collectors;
+import lotto.controller.LottoController;
+import lotto.dto.LottoRequest;
+import lotto.exception.LottoInputException;
+import lotto.service.LottoService;
+import lotto.utility.StringUtility;
+import lotto.validator.LottoBuyMoneyValidator;
+import lotto.validator.LottoWinningNumberValidator;
+import lotto.view.InputView;
+import lotto.view.OutputView;
+
 public class Application {
+
+    private final static String LOTTO_SPLITTER = ",";
+
+    private static int buyMoney = -1;
+    private static List<Integer> winningNumbers = null;
+    private static int bonusNumber = -1;
+
+
     public static void main(String[] args) {
-        // TODO: 프로그램 구현
+        LottoRequest lottoRequest = null;
+        while (lottoRequest == null) {
+            lottoRequest = inputLottoRequestDto();
+        }
+        LottoController lottoController = new LottoController(new LottoService());
+        lottoController.buyLottos(lottoRequest);
+    }
+
+    private static LottoRequest inputLottoRequestDto() {
+        try {
+            buyMoney = inputBuyMoney();
+            winningNumbers = inputWinningNumbers();
+            bonusNumber = inputBonusNumber();
+        }catch(LottoInputException e){
+            OutputView.printError(e);
+            return null;
+        }
+        return new LottoRequest(buyMoney,winningNumbers,bonusNumber);
+    }
+
+    private static int inputBuyMoney() {
+        if(buyMoney != -1) return buyMoney;
```

Comment:
- `-1`은 무엇을 의미하나요?
- 띄어쓰기, 중괄호 컨벤션을 지켜 주세요 😢 
```suggestion
        if (buyMoney != -1) {
            return buyMoney;
        }
```

Reply:
-1은 초기값을 의미하는 것이었는데요, 이제보니 상수화 빼먹어서 의미가 명확하게 전달되지 않았던것 같아요..
다음에는 상수화 빼먹지않고 해볼게요!

컨벤션을 지킬려고 노력하는 중에도 가끔 옛날 습관이 나오는 것 같아요 ㅠㅠ
두번 확인습관도 들여야겠어요